xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-code/src/main/java/org/xwiki/rendering/code/layout/CodeLayoutHandler.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rendering.code.layout;
+
+import java.util.List;
+
+import org.xwiki.component.annotation.Role;
+import org.xwiki.rendering.block.Block;
+
+/**
+ * Arranges a list of code rendering blocks into a specific layout.
+ * 
+ * @version $Id$
+ * @since 11.3
```

Comment:
You have to change this: the changes for 11.3 are fixed now that 11.3RC1 has been released. So your change might be only applied to 11.4RC1 now.

Reply:
Also consider using the `@Unstable` annotation since it's a new API

Reply:
Will do ASAP.